skytemple_files/common/ppmdu_config/script_data.py

```diff
@@ -299,14 +300,7 @@ def __init__(self, ssa_id: int, name: str, ssb_id: int | None = None):
 
     @property
     def id(self) -> int:
-        """
-        For backwards compatibility.
-        """
-        warnings.warn(
-            DeprecatedToBeRemovedWarning("Please use self.ssa_id instead.", (2, 0, 0)),
-            stacklevel=2,
-        )
-        return self.ssa_id
+        return self.ssb_id
```

Comment:
I'm not sure if this change is safe, but it'd allow us to get rid of `SsbScriptDirection`, which saves a type check.

Reply:
I don't really get this change, can't you just use `ssa_id` or `ssb_id`? If anything this should either be removed or kept because there is still code expecting this to return `ssa_id`, in which case everything would break.

Reply:
oh you need it so it's compatible with SsbScriptDirection, Let me check the code...

Reply:
Nah, we really shouldn't just change it like this, it's too risky... Maybe there's still ways to remove `SsbScriptDirection` but not use `Pmd2ScriptDirection.id` either

Reply:
I found another way that should work.